### PR TITLE
Issue#1375 typography

### LINKED ISSUE #1375: Finalize typos
UX + UI + Dev discussion to finalize and then implement

### PR DESCRIPTION
**Pull request by jenna1k**
resolve #1375 

sync typography with zeplin
light-theme:
https://app.zeplin.io/project/5ca741b3ed21e79faa266664/screen/5e061879ead7fd17a5bb2aac
dark-theme:
https://app.zeplin.io/project/5de7944cf85691577aacfbb7/screen/5e1305353cbeb61b6ec84d7a

### CODE REVIEW

File: styles/components/bf-font-color.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+.light-theme {
```
Comment (Mussuranae) [inline styles/components/bf-font-color.scss]: I think you can write a mixin for the color typo :

```
@mixin bf-color-font($theme) {
  $color-font: map-get($theme, font);

  .mat-display-4 {
     color: mat-color($color-font, display-4);
  }
}
```

and in the dark/light.scss, you add :
``` 
$bf-dark/light-theme: (
  // Standard
  primary: $bf-light-primary,
  accent: $bf-light-accent,
  warn: $bf-light-warn,
  is-dark: false,
  foreground: $bf-light-foreground,
  background: $bf-light-background,
  font:  $bf-dark/light-font,
); 
```

Comment (jenna1k) [reply styles/components/bf-font-color.scss]: I created mixin for bf-font-color

File: styles/theme.scss [hunk 2]
```diff
@@ -16,6 +16,10 @@
 @import 'footer';
 @import './components/bf-widget.scss';
 @import './components/mat-sidenav.scss';
+@import './components/bf-font-color.scss';
+
+@include angular-material-typography($bf-typography);
+@include mat-core($bf-typography);
```
Comment (GrandSchtroumpf) [inline styles/theme.scss]: Don't duplicate. It's either `angular-material-typography($bf-typography)` OR `mat-core($bf-typography)`.

File: styles/themes/dark.scss [hunk 3]
```diff
@@ -147,5 +165,6 @@ $bf-dark-theme: (
   warn: $bf-dark-warn,
   is-dark: true,
   foreground: $bf-dark-foreground,
-  background: $bf-dark-background
+  background: $bf-dark-background,
+  font: $bf-dark-font
```
Comment (GrandSchtroumpf) [inline styles/themes/dark.scss]: Careful, `font` is not a key standard to the angular theme : 
https://github.com/angular/components/blob/master/src/material/core/theming/_theming.scss#L69

Add a comment to precise it for future developers reading your code.

File: styles/components/bf-font-color.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (jenna1k) [reply styles/components/bf-font-color.scss]: I've already created mixin in the last commit

File: styles/theme.scss (hunk 2 above)
Comment (jenna1k) [reply styles/theme.scss]: removed `angular-material-typography($bf-typography)`

File: styles/themes/dark.scss (hunk 3 above)
Comment (jenna1k) [reply styles/themes/dark.scss]: added comments

**Pull request merged**